jdk/test/java/lang/VirtualThread/VirtualThreadPinnedEventThrows.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
```

Comment:
Although it doesn't matter on running this test, it would be better to use `/*` due to this is not Javadoc.

Reply:
Sure, I have made the changes. The example given in the jtreg introduction documentation (https://openjdk.org/projects/code-tools/jtreg/intro.html) does indeed use regular comments, but I found that the source files contain /** (documentation comments). Which one should I follow?

Reply:
> but I found that the source files contain /** (documentation comments). Which one should I follow?

I just think it should use `/*` for jtreg test description.
As for other comments/Javadocs using`/**`, I have no suggestion. You can keep them as that.